docs/package.json

```diff
@@ -53,7 +53,8 @@
     "remark-mdx-images": "^1.0.2",
     "rimraf": "^3.0.2",
     "sass": "^1.43.4",
-    "slugify": "^1.6.3"
+    "slugify": "^1.6.3",
+    "style-dictionary": "3.9.1"
```

Comment:
Do the docs also rely on `style-dictionary`? Seems like we wouldn't have been able to deploy without this if so?

Reply:
`style-dictionary` referenced in docs/src/components/home/themes/index.preval.ts and not included in docs/package.json

We didn't directly observe a failure though